docs/data-sources/lb_acls.md

```diff
@@ -23,33 +25,33 @@ data "scaleway_lb_acls" "byFrontID_and_name" {
 
 ## Argument Reference
 
-- `frontend_id` - (Required) The frontend ID this ACL is attached to. ACLs with a frontend ID like it are listed.
-  ~> **Important:** LB Frontends' IDs are [zoned](../guides/regions_and_zones.md#resource-ids), which means they are of the form `{zone}/{id}`, e.g. `fr-par-1/11111111-1111-1111-1111-111111111111`
+- `frontend_id` - (Required) The frontend ID this ACL is attached to. ACLs with a matching frontend ID are listed.
```

Comment:
@yfodil I changed this to "matching" everywhere because it's clearer, is it OK? Does the filter work on an exact match or a partial/similar match?

Reply:
For UUIDs its an exact match